iree/turbine/kernel/wave/index_sequence_analysis.py

```diff
@@ -303,7 +319,9 @@ def has_gpr_offsets(node: fx.Node) -> bool:
                 )
                 reshape.expanded_dims = custom.expanded_dims
                 reshape.vector_shapes = custom.vector_shapes
+                reshape.index = custom.index
```

Comment:
Just curious, in what case would we need this?

Reply:
I'm using index to determine if dynamic val was partitioned or not: https://github.com/iree-org/iree-turbine/pull/511/files#diff-42e6ce65d001220b0098fcfb15f4c93f181a3f53d461b2ce97c2d27d254b5cb4R267

This is ugly, but I didn't found a better way.

Reply:
> detect if op was part of `gpr_offset` partition
Just curious, how/where are we doing this actual check

Reply:
```
                        if any(
                            sympy.sympify(v.start).has(GPR_NUM)
                            for v in get_custom(dyn_val).index.values()
                        ):
```

Reply:
Doesn't this check come before the `reshape.index = custom.index` ?

Reply:
This check is for there read op, but `reshape.index = custom.index` will happen for the `self_index` ops before the read.

Reply:
Ahh I see makes sense, thanks!